.github/workflows/build.yml

```diff
@@ -229,8 +229,6 @@ jobs:
     permissions:
       # required to write sarif report
       security-events: write
-      # required to check out the repository
-      contents: read
```

Comment:
Will this remove the `contents: read` ? The way I understood it from their docs (but need to re-read) is that setting _any_ permissions resets all the other ones already set; https://github.com/docker/buildx/blob/704b2cc52dea1f2d2d7540d119144296ef520a17/.github/workflows/build.yml#L3-L10

In this case (my interpretation) it would mean that it only has `security-events: write`, but no longer `contents: read`.

Reply:
Jobs inherit from global permissions set for the workflow, hence why we got this error when trying to create a GitHub Release: https://github.com/docker/buildx/pull/2728#issuecomment-2443587745